conftest.py

```diff
@@ -109,8 +110,8 @@ def test_run(project_root, request) -> str:
     return test_results_dir
 
 
-@pytest.fixture(scope="function")
-def py_config(project_root, request) -> PyleniumConfig:
+@pytest.fixture(scope="session")
+def _py_config(project_root, request) -> PyleniumConfig:
     """Initialize a PyleniumConfig for each test
 
     1. This starts by deserializing the user-created pylenium.json from the Project Root.
```

Comment:
I guess I should also adapt the documentation here and add some to `py_config`.

Reply:
Yes, please! That would be helpful, but I like the deepcopy approach 💪🏽

Reply:
Done.